pkg/applier/kpt_applier.go

```diff
@@ -525,7 +525,9 @@ func (a *Applier) sync(ctx context.Context, objs []client.Object) (map[schema.Gr
 // Errors implements Interface.
 // Errors returns the errors encountered during apply.
 func (a *Applier) Errors() status.MultiError {
-	return a.errs
+	// TODO: Make read/write of a.errs thread-safe
```

Comment:
Unfortunately, we can't just add a mutex around the errors here, because they're modified in multiple places. But fixing that is a larger change I want to do in a separate PR.